packages/api/test/functional/technology-cost.spec.js

```diff
@@ -211,3 +223,65 @@ test('PUT /technologies/:id/costs deletes costs with empty cost array.', async (
 		costs: [],
 	});
 });
+
+test('PUT /technologies/:id/costs calls algoliasearch.saveObject with implementation and maintenance costs if it is provided', async ({
+	assert,
+	client,
+}) => {
+	const defaultTermFem = 'Não definida';
+	const defaultTermMasc = 'Não definido';
+
+	const lastTechnology = await Technology.last();
+	const technologyCostInst = await lastTechnology.technologyCosts().first();
+
+	const loggeduser = await User.create(researcherUser);
+	await lastTechnology.users().attach([loggeduser.id]);
+
+	await technologyCostInst.load('costs');
+
+	const updatedTechnologyCost = technologyCostInst.toJSON();
+
+	const newImplementationCost = {
+		cost_type: 'implementation_costs',
+		description: 'Custo de implantação adicional',
+		type: 'material',
+		quantity: 2,
+		value: 10000,
+	};
+
+	const newMaintenanceCost = {
+		cost_type: 'maintenance_costs',
+		description: 'Custo de manutenção adicional',
+		type: 'material',
+		quantity: 1,
+		value: 500,
+	};
+
+	updatedTechnologyCost.costs = [];
+	updatedTechnologyCost.costs.push(newImplementationCost);
+	updatedTechnologyCost.costs.push(newMaintenanceCost);
```

Comment:
```suggestion
	updatedTechnologyCost.costs = [newImplementationCost, newMaintenanceCost];
```

Reply:
@lcnogueira I investigated but couldn't find a reason to tests breaking after this change.. Any thoughts?

Reply:
Well, passing the consts directly to the array didn't work, but passing the object works, so I did this way 👍

Reply:
Weird, right? But it works...this is what matters 😄